rmf_fleet_adapter/src/rmf_fleet_adapter/events/GoToPlace.cpp

```diff
@@ -402,7 +467,7 @@ std::string wp_name(const agv::RobotContext& context)
 
 //==============================================================================
 std::optional<rmf_traffic::agv::Plan::Goal> GoToPlace::Active::_choose_goal(
-  bool only_same_map) const
+  bool only_same_map)
 {
   auto current_location = _context->location();
   auto graph = _context->navigation_graph();
```

Comment:
It should have been part of the original PR but I believe this would also create a copy of the whole robot's navgraph, which can be quite significant. We should probably do `const auto&` variables instead also to make sure we don't edit them by accident